src/pivot-table/hooks/__tests__/use-table-rect.test.ts

```diff
@@ -34,13 +35,13 @@ describe("use-table-rect", () => {
   test("should consider pagination height", () => {
     shouldShowPagination = true;
     const { height } = renderer();
-    expect(height).toBe(rect.height - PAGINATION_HEIGHT);
+    expect(height).toBe(rect.height - PAGINATION_HEIGHT - 1);
```

Comment:
why is that extra 1 px?
is it a border somewhere in pagination? is so, i think it's better to use the exported value of `PAGINATION_FOOTER_BORDER` instead

Reply:
Yes that 1 is the border. Will fix and use const value instead